edd-terms-per-product.php

```diff
@@ -24,67 +27,89 @@ function __construct() {
 
 	}
 
-
+	/**
+	 * Loads the text domain
+	 */
 	public function load_textdomain() {
 		// Set filter for plugin's languages directory
 		$lang_dir = dirname( plugin_basename( __FILE__ ) ) . '/languages/';
 		$lang_dir = apply_filters( 'edd_terms_per_product_lang_directory', $lang_dir );
 
 		// Traditional WordPress plugin locale filter
-		$locale        = apply_filters( 'plugin_locale',  get_locale(), 'edd-terms-per-product' );
-		$mofile        = sprintf( '%1$s-%2$s.mo', 'edd-terms-per-product', $locale );
+		$locale = apply_filters( 'plugin_locale', get_locale(), 'edd-terms-per-product' );
+		$mofile = sprintf( '%1$s-%2$s.mo', 'edd-terms-per-product', $locale );
 
 		// Setup paths to current locale file
 		$mofile_local  = $lang_dir . $mofile;
 		$mofile_global = WP_LANG_DIR . '/edd-terms-per-product/' . $mofile;
 
 		if ( file_exists( $mofile_global ) ) {
-			// Look in global /wp-content/languages/edd folder
+			// Look in global /wp-content/languages/edd-terms-per-product folder
 			load_textdomain( 'edd-terms-per-product', $mofile_global );
 		} elseif ( file_exists( $mofile_local ) ) {
-			// Look in local /wp-content/plugins/easy-digital-downloads/languages/ folder
+			// Look in local /wp-content/plugins/edd-terms-per-product/languages/ folder
 			load_textdomain( 'edd-terms-per-product', $mofile_local );
 		} else {
 			// Load the default language files
 			load_plugin_textdomain( 'edd-terms-per-product', false, $lang_dir );
 		}
 	}
 
-
+	/**
+	 * Renders the download meta box.
+	 *
+	 * @param int $post_id
+	 */
 	public function metabox_field( $post_id = 0 ) {
 		$terms = get_post_meta( $post_id, '_edd_download_terms', true );
 		?>
 		<p>
 			<strong><?php printf( esc_html__( '%s Terms of Use:', 'edd-terms-per-product' ), esc_html( edd_get_label_singular() ) ); ?></strong>
 		</p>
 		<p>
-			<textarea name="_edd_download_terms" id="edd_download_terms" rows="10" cols="50" class="large-text"><?php echo $terms; ?></textarea>
+			<textarea name="_edd_download_terms" id="edd_download_terms" rows="10" cols="50" class="large-text"><?php echo esc_textarea( $terms ); ?></textarea>
 			<label for="edd_download_terms"><?php printf( esc_html__( 'Enter the terms of use for this %s:', 'edd-terms-per-product' ), edd_get_label_singular() ); ?></label>
-
 		</p>
 		<?php
 	}
 
+	/**
+	 * Adds our meta key to the list of fields to save.
+	 *
+	 * @param array $fields
+	 *
+	 * @return array
+	 */
 	public function fields_to_save( $fields = array() ) {
 		$fields[] = '_edd_download_terms';
+
 		return $fields;
 	}
 
+	/**
+	 * Sanitizes the terms of use before saving.
+	 *
+	 * @param string $data
+	 *
+	 * @return string
+	 */
 	public function sanitize_terms_save( $data ) {
 		return wp_kses( $data, array(
```

Comment:
```suggestion
		return wp_kses(
			$data,
			array(
```

Minor thing but moving these to their own lines makes my sniffer a lot happier with the added tabs.

Reply:
I don't like it, but I'll concede. :joy: